src/java/org/apache/cassandra/schema/MigrationCoordinator.java

```diff
@@ -180,6 +199,18 @@ boolean wasReceived()
         {
             return receivedSchema;
         }
+
+        @Override
+        public String toString()
```

Comment:
Do we actually use this one anywhere?

Reply:
in log messages

Reply:
Yes, but I didn't see it used, probably I missed it :-)

Reply:
we do not have to call `.toString()` explicitly in statements like `log.info("bla bla bla {}", info);` :)